src/STEPConstruct/STEPConstruct_Styles.cxx

```diff
@@ -71,6 +71,176 @@
 #include <StepVisual_ContextDependentOverRidingStyledItem.hxx>
 #include <StepShape_ShapeRepresentation.hxx>
 
+namespace
+{
+  //=======================================================================
+  //function : ProcessAsSurfaceStyleRendering
+  //purpose  : Process StepVisual_SurfaceStyleElementSelect to extract a
+  //           render color and render trnasparency from it. Returns true,
+  //           if theSSES was of type StepVisual_SurfaceStyleRendering
+  //           (even if color and transparency data couldn't be extracted
+  //           for some reason), otherwise returns false.
+  //=======================================================================
+  Standard_Boolean ProcessAsSurfaceStyleRendering (
+    const StepVisual_SurfaceStyleElementSelect& theSSES,
+    Handle (StepVisual_Colour)&                 theRenderColour,
```

Comment:
I guess Handle_( is not the best option.
I mean Handle_space_( probably needs to avoid that. anything below